src/README.md

```diff
@@ -0,0 +1,499 @@
+# Maynard
+
+Functionality to generate transition matrices, absorbing states, and other
+results for the heterogeneous moran process.
+
+## Tutorial
+
+In this tutorial we will see how to use `maynard` in order to generate the
+transition matrix defining a heterogeneous public goods game. For background
+information on Markov processes in general we recommend:
+Probability, Markov Chains, Queues, and Simulation by
+William J Stewart
+
+We begin by defining a state space. This is the set of possible populations for
+a N players with k actions, and can be obtained by the following code. In this
+example, we shall use N=3 and k=2
+
+```python
+import main
+
+N = 3
+k = 2
+
+state_space = main.get_state_space(N=N, k=k)
```

Comment:
Use doctests for all the code in documentation (https://vknight.org/pfm/building-tools/07-testing/how/main.html#how-to-format-doc-tests). Add running that to CI (see Nashpy's CI for an example):